### ISSUE TITLE
Add context for when Output is Focused / Visible

### ISSUE THREAD

**Issue by 3v1n0** [Microsoft/vscode#44372]
I wanted to customize my keybindings depending on the OutputPanel state, but it looks there's no clause to set in the `when` stanza for it.

**Comment by isidorn** [comment]
There is a context `inOutput`, please try that

**Comment by isidorn** [comment]
We accept PRs that update our docs. Checkout the vscode-docs repo.
No this is not currently supported, we accept PRs that add this.

Code pointers
Similar to the viewlet service here
https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/services/viewlet/browser/viewletService.ts#L19
The panelPart would add a context key https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/browser/parts/panel/panelPart.ts#L34

**Comment by roblourens** [comment]
`"when": "activePanel == 'workbench.panel.repl'"` works but how does a user learn the panel ID?

**Comment by isidorn** [comment]
@roblourens good point. However we already had that issue with the viewlets, since we already have a "activeViewlet" conext key.
We should document this properly @gregvanl
